octobot/constants.py

```diff
@@ -46,20 +47,24 @@
 ENV_TENTACLES_URL_TAG = "TENTACLES_URL_TAG"
 ENV_TENTACLES_URL_CATEGORY = "TENTACLES_URL_CATEGORY"
 ENV_COMPILED_TENTACLES_URL_CATEGORY = "COMPILED_TENTACLES_URL_CATEGORY"
+ENV_TENTACLES_PACKAGES_URL = "TENTACLES_PACKAGES_URL"
+ENV_COMPILED_TENTACLES_PACKAGES_URL = "COMPILED_TENTACLES_PACKAGES_URL"
 ENV_TENTACLES_URL_SUBCATEGORY = "TENTACLES_URL_SUBCATEGORY"
 ENV_COMPILED_TENTACLES_URL_SUBCATEGORY = "COMPILED_TENTACLES_URL_SUBCATEGORY"
 TENTACLES_REQUIRED_VERSION = f"{os.getenv(ENV_TENTACLES_URL_TAG, LONG_VERSION)}"
 DEFAULT_TENTACLES_URL = os.getenv(
     ENV_TENTACLES_URL,
     f"{OCTOBOT_ONLINE}/{REPOSITORY}/{TENTACLES_REPOSITORY}/{OFFICIALS}/"
     f"{os.getenv(ENV_TENTACLES_URL_CATEGORY, TENTACLE_BASE_CATEGORY)}/"
+    f"{os.getenv(ENV_TENTACLES_PACKAGES_URL, TENTACLE_PACKAGES)}/"
```

Comment:
If `ENV_TENTACLES_PACKAGES_URL` is not an url (www.xyz.com), I think we should name it without this _url suffix, maybe `ENV_TENTACLES_PACKAGES_TYPE` ?

Reply:
Good idea :+1: